packages/ui-app/src/util/formatBalance.ts

```diff
@@ -29,7 +30,7 @@ const SI: Array<SiDef> = [
   { power: -9, value: 'n', text: 'nano' },
   { power: -6, value: 'µ', text: 'micro' },
   { power: -3, value: 'm', text: 'milli' },
-  { power: 0, value: '-', text: '----' }, // position 8
+  { power: 0, value: '-', text: 'Unit' }, // position 8
   { power: 3, value: 'k', text: 'Kilo' },
```

Comment:
This should get the 'more than unsubstantial' label, but is there a reason why Kilo is the only unit larger than X / DOT where the token symbol is lowercase (k), all the others are uppercase (M, G, etc). I was asking myself the same question this morning, when I was looking into this issue...

Reply:
https://en.wikipedia.org/wiki/International_System_of_Units#Prefixes

Reply:
Thanks!